framework/packages/abstract-client/Cargo.toml

```diff
@@ -13,7 +13,7 @@ keywords.workspace = true
 
 [features]
 default = ["test-utils"]
-test-utils = ["cw-asset"]
+test-utils = ["cw-asset", "cw20", "cw20-base", "cw-plus-interface"]
```

Comment:
I thought optional deps require a `deps:` prefix? so `"deps:cw20", "deps:cw20-base", ...` 
@Buckram123 do you know? 

I think adding them this way also exposes them as their own features. 

So `features = ["cw20"]` is something you can do when importing `abstract-client`, but again could be wrong!

Reply:
I don't think we should have partial features like that since just a dep on `cw20` would not be sufficient for using the `cw20_builder`. But having a feature for the `cw20_builder` as a whole is a good idea.